cli/exec.go

```diff
@@ -53,6 +64,10 @@ func ConfigureExecCommand(app *kingpin.Application) {
 		Short('m').
 		StringVar(&input.MfaToken)
 
+	cmd.Flag("json", "AWS credential helper").
```

Comment:
https://www.google.com/search?hl=en&q=%22aws%20credential%20helper%22 yields nothing of use, might there be a clearer way to describe this?